src/sqlfluff/rules/L040.py

```diff
@@ -29,8 +30,9 @@ class Rule_L040(Rule_L010):
             false
         from foo
 
-    | **Best practice**
-    | Ensure all literal null/true/false literals cases are used consistently
+    **Best practice**
+
+    Ensure all literal ``null``/``true``/``false`` literals cases are used consistently
```

Comment:
The wording here is weird.
```suggestion
    Ensure all literal ``null``/``true``/``false`` literals are consistently upper or lower case
```